editor/components/post-excerpt/style.scss

```diff
@@ -1,3 +1,7 @@
+.editor-post-excerpt > .blocks-base-control {
+	margin-bottom: 0;
```

Comment:
Should this be specific to EditorPostExcerpt? Seems more like an issue with all TextareaControl that they have too much margin bottom (textarea has its own padding in addition to base control)